agent/replacecall/src/main/resources/default-load-exclusions.txt

```diff
@@ -0,0 +1,21 @@
+// This file contains default package exclusions for the replacecall agent.
```

Comment:
No, I don't.  (That's why I wrote the comment.)
The relationship between loading and transforming is an implementation detail.  It will be confusing to people who have read the manual and then see the file name.  It's best to keep names focused on specifications or behavior rather than implementation.